### PR TITLE
Update and add NPC movement scripts - Elwynn Forest

### PR DESCRIPTION
**Pull request by NickTyrer**
This PR adds some new movement scripts and updates existing ones using waypoints from sniffs where possible.

### CODE REVIEW

File: sql/migrations/20200627204553_world.sql [hunk 1]
```diff
@@ -8,7 +8,7 @@ IF v=0 THEN
 INSERT INTO `migrations` VALUES ('20200627204553');
 -- Add your query below.
 
-
+--forests edge
```
Comment (ratkosrb) [inline sql/migrations/20200627204553_world.sql]: missing space after --

File: sql/migrations/20200627204553_world.sql [hunk 2]
```diff
@@ -279,6 +279,78 @@ INSERT INTO `creature_movement` (`id`, `point`, `position_x`, `position_y`, `pos
...
+UPDATE `creature` SET `position_x` = -9501.6, `position_y` = -1282.86, `position_z` = 43.6379, `orientation` = 6.01566 WHERE `guid` = 81349;
+UPDATE `creature` SET `position_x` = -9523.21, `position_y` = -1290.25, `position_z` = 44.1174, `orientation` = 5.23877 WHERE `guid` = 81250;
+
+
+
+
+
+
+-- eastvale peasant 81252
+UPDATE `creature_template` SET `ai_name` = "EventAI" WHERE entry = 11328;
```
Comment (ratkosrb) [inline sql/migrations/20200627204553_world.sql]: Put `` around entry, also use ' for string instead of ".

**Comment by NickTyrer** [comment]
TODO
- Should the lumberjacks at northshire abbey move or be static?

File: sql/migrations/20200627204553_world.sql [hunk 3]
```diff
@@ -218,18 +218,23 @@ INSERT INTO `creature_movement_special` (`id`, `point`, `position_x`, `position_
...
 (58302, 9, -9300.58, 404.784, 74.2369, 100, 2000, 0, 0);
 
--- add stormwind guard 79861 waypoints
+
+-- add missing guard
+INSERT INTO `creature` (`guid`, `id`, `id2`, `id3`, `id4`, `map`, `display_id`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `wander_distance`, `health_percent`, `mana_percent`, `movement_type`, `spawn_flags`, `visibility_mod`, `patch_min`, `patch_max`) VALUES
+(99140, 1423, 0, 0, 0, 0, 0, 0, -9073.3, 427.18, 93.0558, 0.887339, 285, 285, 0, 100, 0, 2, 0, 0, 0, 10);
+
+-- add stormwind guard 99140 waypoints
 UPDATE `creature` SET `position_x` = -9073.3, `position_y` = 427.18, `position_z` = 93.0558, `movement_type` = 2 WHERE `guid` = 79861;
```
Comment (ratkosrb) [inline sql/migrations/20200627204553_world.sql]: You changed guid in waypoints below, but not in update here. Is that intended?

Comment (NickTyrer) [reply sql/migrations/20200627204553_world.sql]: Not intended, my mistake cheers for reviewing.

**Pull request merged**